README.md

```diff
@@ -76,7 +76,26 @@ Serverless: GET /api
 Serverless: Offline listening on http://localhost:4000
 ```
 
+### Testing
+
+### Integration tests
+
+Integration tests live in test. We just Jest for integration tests.
+
+#### Unit tests
+
+Unit tests live close to the code they tests, in the same directory and file name, ie: `foo.test.js` tests `foo.js`.
```

Comment:
@Bouncey I noticed you have `directive.test.js` instead of `directives.test.js`. Was that intentional?

Reply:
Nope, should be directives 👍

Reply:
#95 refers